frontend/src/components/place/AddPlace.vue

```diff
@@ -445,63 +470,58 @@ export default {
                 phone: '',
                 twitter: '',
                 facebook: '',
-                instagram: ''
+                instagram: '',
+                worktime: {
+                    'mo': {
+                        start:  new Date(0,0,0,10),
```

Comment:
Use [moment.js](https://momentjs.com/) instead of new Date().

Reply:
Done.